test/e2e/far_e2e_test.go

```diff
@@ -303,7 +325,7 @@ func createFAR(nodeName string, agent string, sharedParameters map[v1alpha1.Para
 			SharedParameters:    sharedParameters,
 			NodeParameters:      nodeParameters,
 			RemediationStrategy: strategy,
-			RetryCount:          5,
+			RetryCount:          10,
```

Comment:
do we need this? IIUC the issue was starting a test pod, not rebooting the node?

Reply:
Yes, the test pod wasn't the only case.